.github/workflows/linux-package.yml

```diff
@@ -0,0 +1,77 @@
+name: Linuax Package
+on:
+  push:
+    tags:
+      - 'opentrack-\d+\.\d+\.\d+'
```

Comment:
Maybe it is okay?

Reply:
There are sometimes suffixes like `-rc1` or `-beta`. Maybe try:
`opentrack-\d+\.\d\S+`

Reply:
These are Git-style globs. So something like: `opentrack-[0-9]*.[0-9]*`.